addons/website/static/src/components/resource_editor/resource_editor.js

```diff
@@ -375,7 +375,20 @@ export class ResourceEditor extends Component {
      */
     async saveXML(resource) {
         const { id, arch } = resource;
-        const context = { ...this.context, lang: false };
+        if (!this.website.currentWebsite.metadata.defaultLang) {
+            // TODO: Remove in master.
+            // Default language was not available in template.
+            const languageRecords = await this.orm.read(
+                "website", [this.website.currentWebsite.id],
+                ["default_lang_id"],
+                { context: this.context }
+            );
+            const defaultLanguageId = languageRecords[0]["default_lang_id"][0];
+            this.website.currentWebsite.metadata.defaultLang = (
+                await this.orm.read("res.lang", [defaultLanguageId], ["code"])
+            )[0]["code"];
+        }
```

Comment:
Replaced the first call by this approach - but I still need the second one because I only get an `id`, and `ir.ui.view.write()` expects a language code in its context.
I thought about supporting `lang: "default"` in the context, like the `/website/lang/<lang>` route... but I do not think I can do this in stable because such a value might be unexpected by customizations.

Reply:
Probably should be a `/save_xml` route instead of a plain `write()` on `ir.ui.view` rather, but ok make sense to keep one

Reply:
@rdeodoo Introduced `/save_xml` route and removed other RPC calls.